dbcollector/knobs.json

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
Delete this file since it's the old format.

Reply:
DONE